features/filesystem/littlefs/littlefs/tests/test_alloc.sh

```diff
@@ -301,5 +301,128 @@ tests/test.py << TEST
     lfs_unmount(&lfs) => 0;
 TEST
 
+echo "--- Outdated lookahead test ---"
```

Comment:
What does this mean in regards to outdated?

Reply:
The test is putting littlefs into the state where the lookahead buffer is filled with files that have been deleted. Making the state of the lookahead buffer outdated.

This happens when you delete files after populating the lookahead buffer.

It's up to the filesystem to recognize this situation and correctly refill the lookahead buffer.

Does that make sense?

Reply:
Gotchya! Yeah that makes sense.